src/Caching/Detector/ChangedFilesDetector.php

```diff
@@ -98,11 +95,6 @@ private function getFilePathCacheKey(string $filePath): string
         return $this->fileHasher->hash($this->resolvePath($filePath));
     }
 
-    private function hashFile(string $filePath): string
```

Comment:
this method did the same thing as `getFilePathCacheKey`, therefore just drop it and save unnecessary re-hash'ing of files, we already hashed before